dnf/base.py

```diff
@@ -119,6 +119,18 @@ def _add_repo_to_sack(self, name):
         self._sack.load_yum_repo(hrepo, build_cache=True, load_filelists=True,
                                  load_presto=repo.deltarpm, load_updateinfo=True)
 
+    @staticmethod
+    def _setup_default_conf():
+        conf = dnf.conf.Conf()
+        subst = conf.substitutions
+        if not 'releasever' in subst:
+            subst['releasever'] = dnf.rpm.detect_releasever('/') or ''
```

Comment:
I think you don't want to use `detect_releasever('/')`. you probably want to use `detect_releasever(conf.installroot)` or something like this